readme.md

```diff
@@ -1,20 +1,70 @@
+<a href="https://terrarium.cldcvr.com/">
+  <img alt="terrarium" src="https://storage.googleapis.com/codepipes-assets/terrarium/assets/T8-logo.png" width="100" />
+</a>
+
+[![Documentation](https://img.shields.io/badge/Documentation-Design_Docs-blue?logo=read-the-docs)](https://terrarium.readme.io/docs/)
+[![Build results](https://github.com/cldcvr/terrarium/actions/workflows/release.yaml/badge.svg)](https://github.com/cldcvr/terrarium/actions/workflows/release.yaml)
+[![GitHub license](https://img.shields.io/github/license/cldcvr/terrarium)](https://github.com/cldcvr/terrarium/blob/main/LICENSE)
+[![Discord](https://img.shields.io/discord/1130781563928444978)](https://discord.gg/gG3gDm9GmF)
+[![Contribute](https://img.shields.io/badge/Contribute-How_to_Contribute-blue?logo=github)](./CONTRIBUTING.md)
+
+---
+
 # Terrarium
 
 The Terrarium project aims to empower platform engineering professionals by providing a comprehensive suite of tools for creating reusable Terraform templates. In the evolving landscape of DevOps, Terrarium aligns with the principles of [Internal Developer Platforms (IDP)](internaldeveloperplatform.org), ensuring streamlined infrastructure provisioning and management.
 
-## Tools
+## Installation Instructions
+
+1. Download [Terrarium](https://github.com/cldcvr/terrarium/releases) and extract the TAR archive.
+
+   ```bash
+   wget https://github.com/cldcvr/terrarium/releases/download/$VERSION/terrarium-$VERSION-linux-amd64.tar.gz
+   ```
+   Example:
+   ```bash
+   wget https://github.com/cldcvr/terrarium/releases/download/v0.3/terrarium-v0.3-darwin-amd64.tar.gz
+   tar -xzf terrarium-v0.3-darwin-amd64.tar.gz
```

Comment:
check the new names (when I changed the build process earlier this week, I removed "darwin" in favour of "macos"